cloudinit/net/ifconfig.py

```diff
@@ -0,0 +1,21 @@
+# This file is part of cloud-init. See LICENSE file for license information.
+
+import copy
+import glob
+import logging
+import os
+import re
+from typing import Optional
+
+from cloudinit import subp, util
+from cloudinit.net import ParserError, renderer, subnet_is_ipv6
+from cloudinit.net.network_state import NetworkState
```

Comment:
The linter should be complaining about lots of unused imports.
Please give it a run and see which ones (most) need to be removed.